languages/php/README.md

```diff
@@ -0,0 +1,100 @@
+# Bitwarden Secrets Manager SDK wrapper for PHP
+
+PHP bindings for interacting with the [Bitwarden Secrets Manager]. This is a beta release and might be missing some functionality.
+Supported are CRUD operations on project and secret entities.
+
+## Installation
+
+Requirements:
+- PHP >= 8.0
+- Composer
+- Bitwarden C libraries which you can generate using BitwardenSDK and following instructions in its readme (requires Rust). https://github.com/bitwarden/sdk
+If you are not using the standalone version of this library, file will be placed in `target/debug` folder if you are using from BitwardenSDK repository.
+- Access token for the Bitwarden account
+
+
+## Usage
+
+To interact with the client first you need to obtain the access token from Bitwarden.
+You can initialize ClientSettings and its setting before passing it to the BitwardenClient.
+
+```php
+$client_settings = new \Bitwarden\Sdk\Schemas\ClientSettings()
+$client_settings->apiUrl = getenv('API_URL') ?: 'https://api.bitwarden.com';
+$client_settings->identityUrl = getenv('IDENTITY_URL') ?: 'https://identity.bitwarden.com';
+$client_settings->userAgent = getenv('USER_AGENT') ?: 'SDK';
+$client_settings->deviceType = getenv('DEVICE_TYPE') ?: 'SDK';
```

Comment:
I made BitwardenSettings class for use instead of client settings. It hanldes default values like provided code sample and handles blank device type.